ui/raidboss/data/05-shb/dungeon/amaurot.js

```diff
@@ -363,9 +363,11 @@
         'Cosmic Shrapnel': '飞散',
         'Cosmic Kiss': '流星坠落',
         'Comet': '彗星',
-        'Burst': '爆炸',
+        'Burst': '大爆炸',
         'Apokalypsis': '启示录',
         'Aetherspike': '以太钉刺',
+        '--sync--': '--sync--',
```

Comment:
Can you remove these, here and elsewhere? In general these are always hidden, and so they shouldn't need to be translated anywhere.  It's not a big deal, but I don't want other people seeing this and then adding it too.

Reply:
removed!